src/screens/LandingScreen.js

```diff
@@ -12,12 +12,16 @@ import { COLOR_PALETTE, NAVIGATORS } from "../utils/constants";
 
 import GlobalStyles from "../utils/GlobalStyles";
 import { setPlaylists } from "../redux/MusicSlice";
+import { getFirebaseUser } from "../utils/firebase";
 
 const LandingScreen = ({ navigation }) => {
 	const dispatch = useDispatch();
+	const user = getFirebaseUser();
 
 	const loadInitialPlaylists = async () => {
-		await auth().signInAnonymously();
+		if (!user) {
```

Comment:
You don't need to check for authenticaiton here. Always grab the playlist data using anon, and then profile updates with the signed in user